app/views/invitation_mailer/invite_with_enroll.html.erb

```diff
@@ -3,28 +3,39 @@
     <%= "Hi #{@salutation_name}," %>
   </p>
   <p>
-  We are happy to inform you that you have been accepted to the Turing School
-  of Software & Design. Congratulations and welcome!
+    Congratulations and welcome! <strong>We are happy to inform you that you have been conditionally accepted to the Turing School of Software & Design.</strong> To join us officially, you will need to successfully complete Mod 0 - learn more <%= link_to("here", "https://mod0.turing.io/") %>.
   </p>
   <p>
-  <strong>Please follow <%= link_to("this link", @url) %></strong> to create an
-  account with Census, the Turing School of Software & Design's identity
-  management application.
+    You are about to embark upon an exciting, challenging, and life-changing experience that will be one of the hardest, best things you've ever done. Turing is hard, but it is worth it. Read some <%= link_to("alumni stories", "https://writing.turing.io/tag/alumni/") %> to learn more. Today, we're inviting you to join our community and enroll!
   </p>
   <p>
-  <strong>Next, please visit <a href='https://enroll.turing.io/'>enroll.turing.io</a></strong> to view enrollment availability, pay your deposit, and pick a starting date.
+    <strong>Your next steps:</strong>
   </p>
+  <ol>
+    <li>
+      <strong>Follow <%= link_to("this link", "http://email.turing.io/wf/click?upn=ZphGLUw808sdEm1ILwvOtBRVLCxNVRDaqd7qDnM2nWhs-2FsFHFqP3treA9dqm6BfXDnSJc1m-2FjQhnupW0Lksbc9Pk3GeP3y5-2BIgrmHpr7cS-2FSqyXFk1HlKAU6WL0OrdD15jddPCYLAzXn2nhdSbruGwvT28l808SlqijM20asKN4-3D_et-2FIvjrxoz0bIAuQSC0hCSSIWpYW1K-2BlkFEivNtVabwVpn03BYLm9Nit1Rv4uSdUDGyAEzg3JfR3IsacBG8lcxpGMXzPrxXbhfyYTAiCncvC73J6fWIOGpc9FGaGLpQaWBeragOfo3gJSfvvAIfOmew38h4-2ByWUWgGZCAZOgMFJI24sJeIp1iud5eZ87D6CllREhlczZhUXc4PudTHHgIw-3D-3D") %></strong> to create an account with Census, the Turing School of Software & Design's identity management application.
```

Comment:
This is a custom jwt link created by Census to allow a user to create an account.

```suggestion
      <strong>Follow <%= link_to("this link", @url) %></strong> to create an account with Census, the Turing School of Software & Design's identity management application.
```